cmd/kube-proxy/app/server_others.go

```diff
@@ -145,7 +129,7 @@ func newProxyServer(
 	var proxier proxy.Provider
 	var detectLocalMode proxyconfigapi.LocalMode
 
-	proxyMode := getProxyMode(string(config.Mode), canUseIPVS)
+	proxyMode := getProxyMode(string(config.Mode))
```

Comment:
should we cleanup and get rid of the internal constants and use the API defined modes?

https://github.com/kubernetes/kubernetes/blob/d5fdf3135e7c99e5f81e67986ae930f6a2ffb047/pkg/proxy/apis/config/types.go#L207-L214